src/pages/home/report/ReportActionItem.js

```diff
@@ -127,6 +129,7 @@ const defaultProps = {
 };
 
 function ReportActionItem(props) {
+    const route = useRoute();
```

Comment:
For performance, I think it's better `useRoute` only once in parent view (report screen.)
And pass only flag to `ReportActionItem`.
It's not performant to subscribe route change on every item.

Reply:
@perunt ⬆️

Reply:
In that case, we will rerender all messages if we want to highlight a message already on the screen.
I'll make changes in a moment.

Reply:
done